Example/Backpack/View Controllers/StarRatingsViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2019. Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import Backpack
+
+class StarRatingsViewController: UIViewController {
+    @IBOutlet weak var starRatingView: StarRating!
+    @IBOutlet weak var largeStarRatingView: StarRating!
+    @IBOutlet weak var ratingSlider: UISlider!
+    override func viewDidLoad() {
+        self.starRatingView.rating = self.ratingSlider.value
```

Comment:
Can we avoid explicitly using `self` when it's not required to resolve ambiquity?

Reply:
Done